### PR TITLE
12/21 update

### PR DESCRIPTION
**Pull request by bxpana**
Change tokenomics to Economics and remove sections about token.

### CODE REVIEW

**Comment by barakshani** [comment]
Why removing this information?

File: docs/faq/economics.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Tokenomics
+# Economics
```
Comment (barakshani) [inline docs/faq/economics.md:1]: I think that "Economics" is not the right word here. I guess we could just remove this entire page.

Comment (bxpana) [reply docs/faq/economics.md:1]: What do you think would be better?

Comment (barakshani) [reply docs/faq/economics.md:1]: Don't really know. This page seems a bit out of context. I suggest to move this text under https://zksync.io/faq/decentralization.html#who-operates-zksync-now, without the question in the title.

**Pull request closed without merging**